.all-contributorsrc

```diff
@@ -2807,7 +2807,17 @@
       "contributions": [
         "code"
       ]
+    },
+    {
+      "login": "viktoriamagyar",
+      "name": "Viktória Magyar",
```

Comment:
Did she contribute? I prefer to use the bot after merging this PR

Reply:
Yes, she was the main organizer of Orchard Harvest 2023. The bot can be used, what I did under https://github.com/OrchardCMS/OrchardCore/pull/15036, but it has policy bot issues under its own PRs so I included that change here.

Reply:
> Yes, she was the main organizer of Orchard Harvest 2023

Sure I never forgot that, but we commonly added contributors if they contribute to the code or docs

Reply:
Perhaps we commonly do that, but the point of All Contributors is specifically to "Recognize All Contributors
Including those that don't push code". That's why they also have the `eventOrganizing` category that I used.